js/widget.js

```
@@ -124,15 +124,54 @@ const render_landscape_h_e = async ({ model, el }) => {
   );
 };
 
+const fetchTechnology = async (base_url) => {
+  const path_parameters_json = `${base_url || ''}/landscape_parameters.json`;
+
+  console.warn(
+    "'technology' will be removed from the Python API in a future release. " +
+      'Please update your code to rely on landscape_parameters.json instead.'
+  );
+
+  try {
+    const response = await fetch(path_parameters_json);
+    if (!response.ok) throw new Error(`HTTP error! status: ${response.status}`);
+    const landscape_parameters = await response.json();
+    return landscape_parameters.technology || 'sst';
+  } catch (e) {
+    console.warn(
```

We can throw an error rather than a console log `throw new Error(`Could not parse technology from ${params}`)`

---

fixed